src/main/kotlin/graphics/scenery/controls/behaviours/VRSelectionWheel.kt

```diff
@@ -0,0 +1,105 @@
+package graphics.scenery.controls.behaviours
+
+import graphics.scenery.Scene
+import graphics.scenery.attribute.spatial.Spatial
+import graphics.scenery.controls.OpenVRHMD
+import graphics.scenery.controls.TrackedDeviceType
+import graphics.scenery.controls.TrackerInput
+import graphics.scenery.controls.TrackerRole
+import graphics.scenery.utils.LazyLogger
+import graphics.scenery.utils.Wiggler
+import org.scijava.ui.behaviour.DragBehaviour
+
+/**
+ * A selection wheel to let the user choose between different actions.
+ *
+ * The list of selectable actions can be changed dynamically.
+ *
+ * @param actions List of named lambdas which can be selected by the user
```

Comment:
For Kotlin Kdoc, the `@param` notation should not be used - it's oldschool javadoc. Use `[nameOfTheParameter]` instead 👍

Reply:
https://kotlinlang.org/docs/kotlin-doc.html#param-name they are using @param here. I don't understand.

Reply:
Okay, disregard. I think I misunderstood the convention. Keep the `@param` version 👍